go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	k8s.io/klog/v2 v2.90.1
 	k8s.io/utils v0.0.0-20230209194617-a36077c30491
 	sigs.k8s.io/cluster-api v1.5.0-rc.0
-	sigs.k8s.io/cluster-api/test v1.5.0-rc.0
+	sigs.k8s.io/cluster-api/test v1.4.4
```

Comment:
@chrischdi Seems like another case of dependabot doesn't handle CAPI correctly

But I wonder why we don't have a replace in CAPV like in CAPA: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/release-2.0/go.mod#L5